common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -291,3 +303,99 @@ def test_clicking_task_history_button_with_success(self):
         self.student_admin_section.set_student_email(self.student_identifier)
         self.student_admin_section.click_task_history_button()
         self.assertTrue(self.student_admin_section.is_background_task_history_table_visible())
+
+
+class DataDownloadsTest(BaseInstructorDashboardTest):
+    """
+    Bok Choy tests for the "Data Downloads" tab.
+    """
+    def setUp(self):
+        super(DataDownloadsTest, self).setUp()
+        self.course_fixture = CourseFixture(**self.course_info).install()
+        self.instructor_username, self.instructor_id = self.log_in_as_instructor()
+        instructor_dashboard_page = self.visit_instructor_dashboard()
+        self.data_download_section = instructor_dashboard_page.select_data_download()
+
+    def verify_report_requested_event(self, report_type):
+        """
+        Verifies that the correct event is emitted when a report is requested.
+        """
+        self.verify_events_of_type(
+            self.instructor_username,
+            u"edx.instructor.report.requested",
+            [{
+                u"report_type": report_type
+            }]
+        )
+
+    def verify_report_downloaded_event(self, report_url):
+        """
+        Verifies that the correct event is emitted when a report is downloaded.
+        """
+        self.verify_events_of_type(
+            self.instructor_username,
+            u"edx.instructor.report.downloaded",
+            [{
+                u"report_url": report_url
+            }]
+        )
+
+    def verify_report_download(self):
+        """
+        Verifies that a report can be downloaded and an event fired.
+        """
+        download_links = self.data_download_section.report_download_links
+        download_links[0].click()
+        expected_url = download_links.attrs('href')[0]
```

Comment:
I think we can verify that it contains the string ``problem_grade_report``. I'll add that too.